projects/demo/src/components/dropdown/index.html

```diff
@@ -0,0 +1,82 @@
+<tr>
+    <td colspan="3">
+        <span tuiTitle>
+            <a
+                tuiLink
+                [routerLink]="routes.Dropdown"
+            >
+                <strong>Dropdown</strong>
+            </a>
+            <span tuiSubtitle>Applied as a host directive</span>
+        </span>
+    </td>
+</tr>
+<tr
+    *ngIf="!hiddenOptions.includes('open')"
+    name="[(tuiDropdownOpen)]"
```

Comment:
This one is called `[(tuiDropdownOpen)]` on its own, when it's a host directive it is aliased as `[(open)]`. I think it will be more often added as a host directive so let's name it `[(open)]`

Reply:
renamed